webrender_traits/src/types.rs

```diff
@@ -439,6 +439,13 @@ pub enum ScrollPolicy {
     Fixed,
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub enum ScrollLocation {
+    Delta(Point2D<f32>), // Scroll by a certain amount.
```

Comment:
please move those comments atop of the elements, preceded by `///` for some real documentation